packages/contracts-bedrock/test/actors/OutputBisectionActors.sol

```diff
@@ -0,0 +1,344 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import { Vm } from "forge-std/Vm.sol";
+
+import { OutputBisectionGame } from "src/dispute/OutputBisectionGame.sol";
+import { IOutputBisectionGame } from "src/dispute/interfaces/IOutputBisectionGame.sol";
+
+import "src/libraries/DisputeTypes.sol";
+
+/// @title GameSolver
+/// @notice The `GameSolver` contract is a contract that can produce an array of available
+///         moves for a given `OutputBisectionGame` contract, from the eyes of an honest
+///         actor. The `GameSolver` does not implement functionality for acting on the `Move`s
+///         it suggests.
+abstract contract GameSolver {
+    /// @notice The HEVM cheatcode address.
+    Vm internal immutable VM;
```

Comment:
Why do this? Can't we import from forge-std's [`TestBase`](https://github.com/foundry-rs/forge-std/blob/master/src/Base.sol#L31) if we specifically want the hevm cheatcode address? Or will future actors use different standard library backends?

Reply:
Good shout - swapped over to this.